File: app/src/main/java/com/timo/dream/ui/activity/projectweb/ProjectWebActivity.java
package com.timo.dream.ui.activity.projectweb;


import android.view.View;

import com.timo.base.BaseConstancts;
import com.timo.base.BaseTools;
import com.timo.base.base.base_webview.CommonWebView;
import com.timo.base.view.TitleBar;
import com.timo.dream.R;
import com.timo.dream.mvp.MVPBaseActivity;

/**
 * MVPPlugin
 * 邮箱 784787081@qq.com
 */

public class ProjectWebActivity extends MVPBaseActivity<ProjectWebContract.View, ProjectWebPresenter> implements ProjectWebContract.View {

    CommonWebView mWebview;
    TitleBar      mTitle;

    @Override
    protected int getContentResId() {
        return R.layout.activity_project;
    }

    @Override
    protected void initEvent() {
        mWebview = findViewById(R.id.webview);
        mTitle = findViewById(R.id.title);
        BaseTools.setTitleBar(mTitle, "项目展示", new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                finish();
            }
        });
        if (getIntent().getSerializableExtra(BaseConstancts.Params) == null) {
            BaseTools.loadWeb(mWebview, "https://fir.im");
        } else {
            BaseTools.loadWeb(mWebview, "https://gitee.com/androidworkmanager/projects");
        }

    }

}

File: app/src/main/java/com/timo/dream/ui/activity/projectweb/ProjectWebActivity.kt
package com.timo.dream.ui.activity.projectweb

import com.alibaba.android.arouter.facade.annotation.Route
import com.timo.base.BaseConstancts
import com.timo.base.BaseTools
import com.timo.base.route.RouteConstant
import com.timo.dream.R
import com.timo.dream.mvp.MVPBaseActivity
import kotlinx.android.synthetic.main.activity_project.*
import kotlinx.android.synthetic.main.title.*

/**
 * MVPPlugin
 * 邮箱 784787081@qq.com
 */
@Route(path = RouteConstant.tag_project)
class ProjectWebActivity : MVPBaseActivity<ProjectWebContract.View?, ProjectWebPresenter?>(), ProjectWebContract.View {
    override fun getContentResId(): Int {
        return R.layout.activity_project
    }

    override fun initEvent() {
        if (intent.getSerializableExtra(BaseConstancts.Params) == null) {
            BaseTools.setTitleBar(view_title, "fir项目展示") { finish() }
            BaseTools.loadWeb(view_webview, "https://fir.im")
        } else {
            BaseTools.setTitleBar(view_title, "码云项目展示") { finish() }
            BaseTools.loadWeb(view_webview, "https://gitee.com/androidworkmanager/projects")
        }
    }
}